source/materials/OpticalMaterialProperties.cc

```diff
@@ -735,6 +735,76 @@ namespace opticalprops {
     return mpt;
   }
 
+  /// PolishedAl ///
+  G4MaterialPropertiesTable* PolishedAl()
+  {
+    G4MaterialPropertiesTable* mpt = new G4MaterialPropertiesTable();
+
+    // REFLECTIVITY
+    // std::vector<G4double> ENERGIES = {
+    //   optPhotMinE_, optPhotMaxE_
+    // };
+    // std::vector<G4double> REFLECTIVITY = {
+    //   1., 1.
+    // };
+
+    std::vector<G4double> ENERGIES = {
+       h_Planck * c_light / (2456.42541 * nm), h_Planck * c_light / (2396.60266 * nm),
+       h_Planck * c_light / (2276.95716 * nm), h_Planck * c_light / (2159.52733 * nm),
+       h_Planck * c_light / (2037.66617 * nm), h_Planck * c_light / (1918.02068 * nm),
+       h_Planck * c_light / (1798.37518 * nm), h_Planck * c_light / (1676.51403 * nm),
+       h_Planck * c_light / (1559.08419 * nm), h_Planck * c_light / (1437.22304 * nm),
+       h_Planck * c_light / (1319.79321 * nm), h_Planck * c_light / (1197.93205 * nm),
+       h_Planck * c_light / (1078.28656 * nm), h_Planck * c_light / (956.42541 * nm),
+       h_Planck * c_light / (838.99557 * nm), h_Planck * c_light / (717.13442 * nm),
+       h_Planck * c_light / (597.48892 * nm), h_Planck * c_light / (477.84343 * nm),
+       h_Planck * c_light / (418.02068 * nm), h_Planck * c_light / (358.19793 * nm),
+       h_Planck * c_light / (293.94387 * nm)
+    };
+    std::vector<G4double> REFLECTIVITY = {
+      .99088, .99082, .98925, .98623, .98611,
+      .98163, .98006, .97849, .97401, .97098,
+      .96941, .96784, .96481, .96033, .96167,
+      .96301, .96289, .96278, .96126, .95830,
+      .94224
+    };
+    mpt->AddProperty("REFLECTIVITY", ENERGIES, REFLECTIVITY);
+
+    // REFLEXION BEHAVIOR
+    std::vector<G4double> ENERGIES_2    = {optPhotMinE_, optPhotMaxE_};
+    std::vector<G4double> ENERGIES_3    = {
```

Comment:
I would move this below, closer to the `rIndex` vector declaration for better readability